### PR TITLE
MOSIP-23715 code fix

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java [hunk 1]
```diff
@@ -443,6 +446,8 @@ public void consumerHealthCheck(Handler<HealthCheckDTO> eventHandler,
 				healthCheckDTO.setEventBusConnected(false);
 				healthCheckDTO.setFailureReason(f.cause().getMessage());
 			}
+			logger.info("Consumer health check ended {} {}",
+					address + " end " + DateUtils.formatToISOString(DateUtils.getUTCCurrentDateTime()));
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java]: we cannot understand the releation between the above 2 logs, we have to add the timestamp to a variable and use it in both places to make the corelation

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java [hunk 2]
```diff
@@ -1,227 +1,239 @@
...
+	/** The mosip queue manager. */
+	@Autowired
+	private MosipQueueManager<MosipQueue, byte[]> mosipQueueManager;
+
+	private static final String PROMETHEUS_ENDPOINT = "/actuator/prometheus";
+
+
+	private static Logger regProcLogger = RegProcessorLogger.getLogger(MosipVerticleAPIManager.class);
+
+	@Value("${registration.processor.healthcheck.timeout}")
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java]: Lets have the default time configured here itself as 5 seconds

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java [hunk 3]
```diff
@@ -435,6 +436,8 @@ private void resumePartition(TopicPartition topicPartition, Promise<Void> promis
 	public void consumerHealthCheck(Handler<HealthCheckDTO> eventHandler,
 			String address) {
 		HealthCheckDTO healthCheckDTO = new HealthCheckDTO();
+		logger.info("Consumer health check started {} {}",
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java]: this and next has to be debug

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java (hunk 1 above)
Comment (sowmya695) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java]: fixed

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java (hunk 3 above)
Comment (sowmya695) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java]: fixed

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java (hunk 2 above)
Comment (sowmya695) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java]: fixed

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**